docs/experiment/general/data-model.md

```diff
@@ -12,35 +12,37 @@ At the top level in Amplitude is your **organization**. Within an organization,
 
 ## Projects
 
-Experiment uses the same projects which are required for Amplitude Analytics. Generally speaking you want to create a project per product and per environment. Since [flags](#flags-and-experiments), [experiments](#flags-and-experiments), and [deployments](#deployments) only exist within a single project, you will want to duplicate these objects across projects for the same product.
```

Comment:
"Experiment uses the same projects which are required for Amplitude Analytics."

I still think this is a valuable piece of information.